pkg/asset/installconfig/aws/metadata.go

```diff
@@ -14,12 +14,16 @@ import (
 type Metadata struct {
 	session           *session.Session
 	availabilityZones []string
+	privateSubnets    map[string]Subnet
+	publicSubnets     map[string]Subnet
 	region            string
+	subnets           []string
+	vpc               string
 	mutex             sync.Mutex
 }
 
 // NewMetadata initializes a new Metadata object.
-func NewMetadata(region string) *Metadata {
+func NewMetadata(region string, subnets []string) *Metadata {
```

Comment:
`subnets` is not used.

Reply:
> `subnets` is not used.

Oops.  Fixed with b98a00d07 -> 80a252193.